themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,36 +28,38 @@
     <label><?=$this->translate('Sort') ?></label>
     <div class="btn-group">
       <?php foreach ($this->sortOptions as $key => $sort): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) ?>" class="btn btn-default js-facet-sort<?php if ($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
+        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) . '&amp;contains=' . urlencode($contains) ?>" class="btn btn-default js-facet-sort<?php if($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>"><?=$this->translate($sort) ?></a>
       <?php endforeach; ?>
     </div>
   </div>
 <?php endif; ?>
-<div class="lightbox-scroll full-facets">
+
+<form>
+  <?php if (method_exists($this->results->getParams(), 'setFacetContains')): ?>
+    <input class="ajax_param" data-name="contains" placeholder="<?=$this->transEscAttr('Filter')?>" class="form-control" type="text" value="<?=htmlspecialchars($this->contains)?>">
+  <?php endif; ?>
+  <?php $ajaxParams = ['facet' => $this->facet, 'facetexclude' => $this->exclude, 'facetop' => $this->operator, 'facetpage' => $this->page, 'facetsort' => $this->sort, 'urlBase' => $urlBase, 'searchAction' => $searchAction, 'ajaxLightbox' => $this->ajaxLightbox ? intval($this->ajaxLightbox) : intval($this->inLightbox)]; ?>
```

Comment:
Could the 'ajaxLightbox' calculation be simplified to `intval($this->ajaxLightbox || $this->inLightbox)`?